src/app_service/service/chat/dto/chat.nim

```diff
@@ -301,13 +303,16 @@ proc toChatDto*(jsonObj: JsonNode): ChatDto =
     result.id = result.communityId & result.id
 
 # To parse Community chats to ChatDto, we need to add the commuity ID and type
-proc toChatDto*(jsonObj: JsonNode, communityId: string): ChatDto =
+proc toChatDto*(jsonObj: JsonNode, communityId: string, communityMembers: seq[ChatMember]): ChatDto =
```

Comment:
The intention behind the communityMembers argument is not clear. Maybe change the name to unpermissionedCommunityMembers or somthing like this?